### ISSUE TITLE
A line only containing a string and a trailing comma should not be reported to exceed max line length

### ISSUE THREAD

**Issue by paul-dingemans** [pinterest/ktlint#2597]
> A line containing a string and a trailing comma should not be reported to exceed max line length as. See example below:
> 
> ```
> throw SomeException(
>     "A long exception message which exceeds the maximum line length and is followed by a (trailing) comma",
>     e,
> )
> // or
> throw SomeException(
>     "A long exception message which exceeds the maximum line length and is followed by a (trailing) comma",
> )
> ```

#2480 does not report string template when it is the only element on the line. But is that string template is followed by a comma, it is still reported.

_Originally posted by @paul-dingemans in https://github.com/pinterest/ktlint/issues/2480#issuecomment-1984318663_
